src/vs/workbench/browser/parts/editor/multiEditorTabsControl.ts

```diff
@@ -904,6 +932,22 @@ export class MultiEditorTabsControl extends EditorTabsControl {
 			EventHelper.stop(e);
 
 			tab.blur();
+
+			if (isMouseEvent(e) && (e.button !== 0 /* middle/right mouse button */ || (isMacintosh && e.ctrlKey /* macOS context menu */))) {
+				if (e.button === 1) {
+					e.preventDefault(); // required to prevent auto-scrolling (https://github.com/microsoft/vscode/issues/16690)
+				}
+
+				return undefined;
```

Comment:
Inconsistent returns, sometimes `undefined` sometimes `void`.

Reply:
This is how it was before, did not know if I would break something if I change it

Reply:
In JS there should be no difference returning `return undefined;` or `return;`